packages/ui-extensions/src/surfaces/admin/extension-points.ts

```diff
@@ -14,6 +14,46 @@ export interface ExtensionPoints {
     CustomerSegmentationTemplateApi<'admin.customers.segmentation-templates.render'>,
     CustomerSegmentationTemplateComponent
   >;
+
+  // Blocks
+  'admin.product-details.block.render': RenderExtension<
+    StandardApi<'admin.product-details.block.render'>,
+    AnyComponent
+  >;
+  'admin.order-details.block.render': RenderExtension<
+    StandardApi<'admin.order-details.block.render'>,
+    AnyComponent
+  >;
+  'admin.customer-details.block.render': RenderExtension<
+    StandardApi<'admin.customer-details.block.render'>,
+    AnyComponent
+  >;
+
+  // Actions
+  'admin.product-details.action.render': RenderExtension<
+    StandardApi<'Playground'>,
```

Comment:
Is `Playground` right here? And the same for the rest of the entries.

Reply:
It's not lol, i goofed it. Will update 🙂

Reply:
Updated